gm4_display_frames/data/gm4_display_frames/functions/potion_tracking/tag_potion.mcfunction

```diff
@@ -0,0 +1,4 @@
+# @s = any potion entity
+# run from #gm4_potion_tracking:tag_potion
+
+execute if predicate gm4_display_frames:invisibility_potion run tag @s add gm4_potion
```

Comment:
This is also a place where the library could copy the data to storage first, and then this function could have 3 commands checking that storage instead of an inefficient predicate.

Reply:
Sorry, I didn't know that predicates were worst. The potion tracking library already puts the entire potion entity data into storage; I just thought the 3x checks would be better in an `alternative` predicate check, thus used the entity data instead of storage data. I'll make the changes to use the storage data.